ml/src/test/scala/ai/h2o/sparkling/ml/models/H2OMOJOModelTestSuite.scala

```diff
@@ -644,6 +644,35 @@ class H2OMOJOModelTestSuite extends FunSuite with SharedH2OTestContext with Matc
     mojo.getCrossValidationModelsScoringHistory().length shouldBe 0
   }
 
+  test("getCoefficients is exposed for trained GLM model") {
+    val model = new H2OGLM()
+      .setSeed(1)
+      .setFeaturesCols("CAPSULE", "RACE", "DPROS", "DCAPS", "PSA", "VOL", "GLEASON")
+      .setLabelCol("AGE")
+      .setLambdaValue(Array(0))
+      .setComputePValues(true)
+      .fit(prostateDataFrame)
+
+    model.getCoefficients().show()
```

Comment:
This line seems like a left over from exploration and can be probably deleted for an automatic test suite .